hack/integration-test-cache.sh

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
+
+# Copyright 2020 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+SCRIPT_ROOT=$(dirname "${BASH_SOURCE}")/..
+source "${SCRIPT_ROOT}/hack/lib/init.sh"
+
+TEMP_DIR=${TMPDIR-/tmp}
+
+cleanup() {
+  rm -rf "${TEMP_DIR}/setup-envtest"
+}
+
+runTests() {
+  kube::log::status "Configuring envtest"
+  source "${TEMP_DIR}/setup-envtest"
+
+  kube::log::status "Running integration test cases"
+  # TODO: make args customizable.
+  go test -timeout=40m -mod=vendor -v \
+	  test/integration/noderesourcetopology_cache_test.go \
+	  test/integration/nrtutils.go \
+	  test/integration/utils.go \
+	  test/integration/main_test.go
```

Comment:
This is already covered in:

https://github.com/kubernetes-sigs/scheduler-plugins/blob/a64627b6d909b80afdc3a51ec6c0f57d94990293/hack/integration-test.sh#L36

Reply:
And `go test` runs against packages instead of files. You have to use `-run TestXYZ` to target the specific tests if they're in the same package.

Reply:
I should probably have looked harder at go docs instead of reinventing my wheel :) I'll just drop these in my next upload.